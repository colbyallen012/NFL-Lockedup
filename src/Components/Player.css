.crime {
  background-color: lightcyan;
  color: black;
  margin: 10px;
  padding: 10px;
  height: 25vh;
  width:  200px;
  /* width: 200px;
  height: 200px; */
}