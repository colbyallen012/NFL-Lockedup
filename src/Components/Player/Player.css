.player {
  background-color: white;
  border-radius: 20px;
  color: black;
  margin: 5px auto;
  text-align: center;
  width:  300px;
}

.deets {
  font-weight: normal;
  font-size: 15px;
}