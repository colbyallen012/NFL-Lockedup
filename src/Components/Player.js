import React from 'react';
import './Player.css';

const Player = ({name, city, team, position, arrests}) => {
  return (
    <div className='crime'>
      <li>{name} {city} {team} {position} Arrest Count: {arrests}</li>
      {/* <h3>{name}</h3>
      <p>{city} {team} {position}</p>
      <p>Arrest Count: {arrests}</p> */}
    </div>
  )
}

export default Player;