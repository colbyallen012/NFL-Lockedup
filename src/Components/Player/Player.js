import React from 'react';
import PropTypes from 'prop-types';
import './Player.css';

const Player = ({name, city, team, position, arrests}) => {
  return (
    <div className='player'>
      <h3 className='crime-player-name'>{name} <span className='deets'>{city} {team} {position}</span></h3>
      <p className='crime-arrests'>Arrest Count: {arrests}</p>
    </div>
  )
}

Player.propTypes = {
  name: PropTypes.string.isRequired,
  city: PropTypes.string.isRequired,
  team: PropTypes.string.isRequired,
  position: PropTypes.string.isRequired,
  arrests: PropTypes.string.isRequired
}

export default Player;