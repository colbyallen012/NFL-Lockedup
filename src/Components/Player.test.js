import React from 'react';
import { shallow } from 'enzyme';
import Player from './Player';

describe('Player', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Player
        name='Colby Allen'
        city='Denver'
        team='Turing'
        position='Front End'
        arrests={0}
      />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})