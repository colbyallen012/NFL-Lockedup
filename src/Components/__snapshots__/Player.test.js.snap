// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player should match the snapshot 1`] = `
<div
  className="position"
>
  <h3>
    Colby Allen
     
    <span
      className="deets"
    >
      Denver
       
      Turing
       
      Front End
    </span>
  </h3>
  <p>
    Arrest Count: 
    0
  </p>
</div>
`;
